### PR TITLE
Integration for osTicket

### PR DESCRIPTION
**Pull request by mskrip**
Toggl button for osTicket system. Can be used for tickets and tasks, layouts for both options is the same. Button is placed in the header of the ticket/task so it always visible.

screenshot from 2017-07-11 12-10-01
__________________________________________________________________________________________
screenshot from 2017-07-11 12-14-43

Timer description is in format: `<ticket/task number> [<ticket/task name>]`

### CODE REVIEW

**Comment by mskrip** [comment]
@syko this one would be greatly appreciated too, thanks

File: src/scripts/origins.json [hunk 1]
```diff
@@ -277,6 +277,11 @@
     "name": "OnlyOffice",
     "clone": "true"
   },
+  "osticket.com": {
+    "url": "*//*.osticket.com/*",
```
Comment (syko) [inline src/scripts/origins.json]: You are missing a `:` here.

File: src/scripts/content/osticket.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  observe: true
+}, function (elem) {
+  var link, description,
+    titleElem = $('.flush-left a'),
+    projectElem = $('.tixTitle'),
+    ticketNameText = titleElem.textContent.trim(),
+    projectNameText = projectElem.textContent.trim();
+
+  description = ticketNameText + " [" + projectNameText + "]";
+  console.log(description);
```
Comment (syko) [inline src/scripts/content/osticket.js]: console.log

File: src/scripts/origins.json (hunk 1 above)
Comment (syko) [inline src/scripts/origins.json]: Also, the default domain for osticket seems to be supportsystem.com no?

**Comment by mskrip** [comment]
@syko Resolved

File: src/scripts/origins.json (hunk 1 above)
Comment (mskrip) [inline src/scripts/origins.json]: Fixed in 59315c8

File: src/scripts/content/osticket.js (hunk 2 above)
Comment (mskrip) [inline src/scripts/content/osticket.js]: Removed in 59315c8

File: src/scripts/origins.json [hunk 3]
```diff
@@ -277,6 +277,11 @@
     "name": "OnlyOffice",
     "clone": "true"
   },
+  "osticket.com": {
+    "url": "*://*supportsystem.com/*",
```
Comment (syko) [inline src/scripts/origins.json]: Ah it appears it the host part needs to have a dot between the wildcard and the domain `*.supportsystem.com`

Comment (mskrip) [inline src/scripts/origins.json]: Yes sorry. Fixed in 54b5252

**Comment by syko** [comment]
That's Integration number 100 if I'm counting correctly! 🙌  🍾 

Thanks for contributing!

**Comment by langri-sha** [comment]
🎊 ❇️ 💯 🙌

**Pull request merged**